expression-language.md

```diff
@@ -182,10 +181,10 @@ The type system contains 3 _primitive_ types:
   32-bit, twos-complement encoding.
 - _Boolean_: A boolean value of "true" or "false".
 
-The [types defined in the CloudEvents specification][ce-type-system] URI, URI Reference and Timestamp are represented as
+The [types defined in the CloudEvents specification][ce-type-system] (URI, URI Reference, and Timestamp) are represented as
```

Comment:
I don't think we need parenthesis here, same for the last comma you added

Reply:
It's a garden-path sentence to have "... defined in the CloudEvents specification URI, ...", since the specification has a URI.

I see though, that this isn't just a reminder of what the 3 types are. It's a rule applying to only those types. I reordered the sentence to get the emphasis right.